hack/local-cmo.sh

```diff
@@ -201,6 +201,9 @@ main() {
 	local operator_config=manifests/0000_50_cluster-monitoring-operator_04-config.yaml
 	local telemetry_conf=/tmp/telemetry-config.yaml
 
+	local desired_version
+	desired_version=$(kubectl get clusterversion version -o json|jq .status.desired.version|sed 's/"//g')
```

Comment:
(nit) we should be able to use jsonpath directly (untested)
```suggestion
	desired_version=$(kubectl get clusterversion version -o jsonpath='{ .status.desired.version }')
```

Reply:
indeed it works :) updated